src/pyscipopt/scip.pxd

```diff
@@ -1608,6 +1633,11 @@ cdef extern from "scip/cons_countsols.h":
     SCIP_RETCODE SCIPcount(SCIP* scip)
     SCIP_RETCODE SCIPsetParamsCountsols(SCIP* scip)
     SCIP_Longint SCIPgetNCountedSols(SCIP* scip, SCIP_Bool* valid)
+    SCIP_RETCODE SCIPincludeConshdlrCountsols   (   SCIP *  scip    )   
+    void SCIPgetCountedSparseSols(SCIP* scip, SCIP_VAR*** vars, int* nvars, SCIP_SPARSESOL*** sols, int* nsols);
+    SCIP_RETCODE SCIPcount  (   SCIP *  scip    );
```

Comment:
Please check for duplicates before commiting.

Reply:
Done.